app/utils/api.ts

```diff
@@ -320,7 +322,7 @@ export const getCSVForRuleRun = async (
       }
     );
 
-    const filename = `${(ruleVersion + "_" + filepath).replace(/\.json$/, ".csv")}`;
+    const filename = `${filepath.replace(/\.json$/, ".csv")}`;
```

Comment:
Did you intend to switch this back to not having a rule version?

Reply:
Forgot to remove the references to ruleversion elsewhere. Name is already too verbose and this was not needed. Updated the IsolationTester and ScenarioCSV references / api calls to remove as well, as we're not passing the actual version anymore.